docs/tools/vdb_table/data/oramasearch.json

```diff
@@ -108,24 +108,24 @@
     "comment": ""
   },
   "pricing": {
-    "value": "free plan / $4.99 per month (pro)",
+    "value": "free plan / $29.99 per month (pro)",
     "source_url": "",
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
```

Comment:
please provide a reference to docs/example notebook for this in "source_url"

Reply:
Orama is written in JavaScript, therefore you can embed it in any JS application on the browser/server. That's well stated in our docs/repo